docs/1_intro.md

```diff
@@ -1,5 +1,7 @@
 # Branch Web SDK
 
+NOTE: Please update to the latest version at your earliest convenience. v2.25.1 patches a critical bug in anticipation of iOS 11.
```

Comment:
"For all users that are on a version earlier than v2.25.0 please update to the latest version. v2.25.0 patches a critical bug in anticipation of iOS 11)